tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1429,7 +1432,7 @@ class CursorRangeInfoConsumer : public SwiftASTConsumer {
 
     ImmutableTextSnapshotRef InputSnap;
     if (auto EditorDoc = Lang.getEditorDocuments()->findByPath(
-            InputFile, /*IsRealpath=*/true))
+            PrimaryFilePath, /*IsRealpath=*/true))
```

Comment:
This is the most significant change. `InputFile` is almost always empty, except in the case of generated buffers (eg. expanded macros). It should have been using the primary file path, which wasn't being passed in. So most of the change is just passing it to all the various callers.